next.config.js

```diff
@@ -88,6 +88,9 @@ const nextConfig = {
       {
         hostname: "og.link3.to",
       },
+      {
```

Comment:
why do we need this? 👀

Reply:
This is the image provider from Farcaster.